src/LibLog.Tests/LibLog.Tests.csproj

```diff
@@ -1,12 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.0;net461;net452</TargetFrameworks>
```

Comment:
xUnit requires net452

Reply:
good catch - restored the test project to net452.